tests/pages/dashboard/ComponentsList-cy.js

```diff
@@ -2,7 +2,8 @@ describe('ComponentsList [10b]', function () {
 
   context('No Components Found [10f]', function () {
 
-    it('shows error when components not found [10d]', function () {
+    // TODO: Turn into unit test
+    xit('shows error when components not found [10d]', function () {
```

Comment:
why is this required to be a unit test? i can see how this could also be an integration test